src/write_gcm.c

```diff
@@ -327,7 +327,8 @@ static int wg_curl_get_or_post(char *response_buffer,
 
   write_ctx.data[0] = 0;
   if (write_ctx.size < 2) {
-    WARNING("write_gcm: The buffer overflowed.");
+    WARNING("write_gcm: The buffer overflowed. Data is %s",
```

Comment:
The usual thing is to just ensure that the messages are the same and then syslog will handle suppressing things down and saying "X has happened 1000 times" (I have this in my logs from the failing to write problem).  One approach I've done is to have the higher level be a generic followed by more details as a DEBUG log message